jetty-util/src/main/java/org/eclipse/jetty/util/component/AbstractLifeCycle.java

```diff
@@ -53,7 +54,7 @@
     public static final String STOPPING = State.STOPPING.toString();
 
     private final List<EventListener> _eventListener = new CopyOnWriteArrayList<>();
-    private final Object _lock = new Object();
```

Comment:
As this is the base-class for many other classes, do we want to consider allowing access to this lock instead of declaring yet another lock in the subclasses?

Reply:
It was a private lock, so lets keep it a private lock.